distribution/src/config/jvm.options

```diff
@@ -120,3 +120,5 @@ ${error.file}
 
 # temporary workaround for C2 bug with JDK 10 on hardware with AVX-512
 10-:-XX:UseAVX=2
+
+-Dlog4j2.loggerContextFactory=org.elasticsearch.common.logging.MarkerLoggerContextFactory
```

Comment:
That will be very problematic in integration tests. Another argument against option3